src/GoTrueClient.ts

```diff
@@ -558,6 +562,27 @@ export default class GoTrueClient {
     }
   }
 
+  private async _handleOpenIDConnectSignIn(
+    { IDToken, nonce, clientID, issuer, provider }: OpenIDConnectCredentials
```

Comment:
```suggestion
    { id_token, nonce, client_id, issuer, provider }: OpenIDConnectCredentials
```

For consistency in naming.

Reply:
It may be that GoTrue expects `IDToken` and `clientID` but in this client, I think `id_token` and `client_id` is more consistent.